src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/MinMax_DT.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.Localization;
+using Microsoft.PowerFx.Core.Types;
+
+namespace Microsoft.PowerFx.Core.Texl.Builtins
+{
+    // Min(arg1:n, arg2:n, ..., argN:n)
```

Comment:
[nit] `Min(arg1:[D|d|T], arg2:[D|d|T], ..., argN:[D|d|T])` - this function takes dates, datetimes and time values, not numbers.